packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1002,11 +1007,7 @@ export class ContainerRuntime
 			idCompressorMode = desiredIdCompressorMode;
 		}
 
-		const createIdCompressorFn = async () => {
-			const { createIdCompressor, deserializeIdCompressor, createSessionId } = await import(
-				"@fluidframework/id-compressor/internal"
-			);
-
+		const createIdCompressorFn = () => {
```

Comment:
Two changes here:

1.  The lambda is no longer async.
2.  The dynamic import has been replaced with the static import above.